oak/server/enclave_server.h

```diff
@@ -143,10 +143,11 @@ class EnclaveServer final : public asylo::TrustedApplication {
 
   void RequestNextCall() {
     // The stream will delete itself when it finishes.
-    auto stream = std::make_shared<grpc_server::GrpcStream>(node_);
+    // auto stream = std::make_shared<grpc_server::GrpcStream>(node_);
+    auto stream = new grpc_server::GrpcStream(node_);
+    auto callback = new std::function<void()>([stream]() { stream->HandleCreateStream(); });
```

Comment:
I believe you can use std::bind to bind the stream

Reply:
I did that at first, but I think lambdas are vaguely more readable? TBH both are quite verbose and not particularly ergonomic.